nightly-playground/lib/nightly-playground-stack.ts

```diff
@@ -26,10 +26,17 @@ export class NightlyPlaygroundStack {
     if (dashboardsUrl === 'undefined') {
       throw new Error('dashboardsUrl parameter cannot be empty! Please provide the OpenSearch-Dashboards distribution URL');
     }
+    const dashboardPassword = scope.node.tryGetContext('dashboardPassword');
```

Comment:
IS this user different from admin user?

Reply:
This is password for kibanaserver user. I kept the username same. Just customized the password.